Public/Src/Tools/Execution.Analyzer/Analyzers.Partner/PipExecutionPerformanceAnalyzer.cs

```diff
@@ -20,13 +20,19 @@ internal partial class Args
         public Analyzer InitializePipExecutionPerformanceAnalyzer()
         {
             string outputFilePath = null;
+            bool isCsvFormat = false;
             foreach (var opt in AnalyzerOptions)
             {
                 if (opt.Name.Equals("outputFile", StringComparison.OrdinalIgnoreCase) ||
                     opt.Name.Equals("o", StringComparison.OrdinalIgnoreCase))
                 {
                     outputFilePath = ParseSingletonPathOption(opt, outputFilePath);
                 }
+                else if (opt.Name.Equals("csv", StringComparison.OrdinalIgnoreCase) ||

```

Comment:
>if [](start = 21, length = 2)

Nit: Why not do the same as some analyzers and look for the extension of the output file i.a. `a.csv` would be a csv file...